src/argilla_server/schemas/v1/workspaces.py

```diff
@@ -29,5 +32,9 @@ class Config:
         orm_mode = True
 
 
+class WorkspaceCreate(BaseModel):
+    name: str = Field(regex=WORKSPACE_NAME_REGEX, min_length=1)
```

Comment:
`regex` has been deprecated since a long time ago.
```suggestion
    name: str = Field(pattern=WORKSPACE_NAME_REGEX, min_length=1)
```

Reply:
I tested like in previous PRs and for some reason it's not working.